eslint-precommit.json

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": [
+    "../mastarm/lib/eslintrc.json"
```

Comment:
This should be changed to point to the node_modules mastarm since the mastarm project isn't necessarily going to exist in an adjacent directory outside of an IBI development environment.

Reply:
Given/pending https://github.com/conveyal/mastarm/pull/310, I have removed this extra eslint settings file in https://github.com/opentripplanner/otp-react-redux/pull/377/commits/e66a45da926abaae948d9460fd49eeee52417e70.